src/node/stereo_panner.rs

```diff
@@ -45,13 +34,56 @@ impl Default for StereoPannerOptions {
     }
 }
 
+/// Assert that the channel count is valid for the StereoPannerNode
+/// see <https://webaudio.github.io/web-audio-api/#audionode-channelcount-constraints>
+///
+/// # Panics
+///
+/// This function panics if given count is greater than 2
+///
+fn assert_valid_channel_count(count: usize) {
```

Comment:
```
#[track_caller]
```

just like the checks in src/lib.rs will give a nicer error message

Reply:
Yup! I forgot this one